website/frontend/src/components/AnalyticsSection/AnalyticsSection.js

```diff
@@ -11,7 +12,7 @@ const AnalyticsSection = () => (
                 <h1>An interactive air quality analytics platform</h1>
                 <div className="AnalyticsSection-info-aside">
                     <p>Access and visualise real-time and historical air quality information across Africa through our easy-to-use air quality analytics dashboard</p>
-                    <a href="https://platform.airqo.net/login" target="_blank" className="AnalyticsSection-info-aside-btn">Learn more {'-->'}</a>
+                    <a href={NETMANAGER_URL} target="_blank" className="AnalyticsSection-info-aside-btn">Learn more {'-->'}</a>
```

Comment:
Users should be routed to platform according to their environment i.e Staging-website -> staging-netmanager, prod-website -> prod-netmanager